server/matchmaker/algorithm/bucket_teams.py

```diff
@@ -21,12 +21,12 @@ class BucketTeamMatchmaker(Matchmaker):
     """
 
     def find(
-        self, searches: Iterable[Search], team_size: int
+            self, searches: Iterable[Search], team_size: int, rating_peak: float
```

Comment:
Don't add extra indents
```suggestion
        self, searches: Iterable[Search], team_size: int, rating_peak: float
```